packages/editor/src/ce/extensions/document-extensions.tsx

```diff
@@ -7,14 +7,16 @@ import { TIssueEmbedConfig } from "@/plane-editor/types";
 import { TExtensions, TUserDetails } from "@/types";
 
 type Props = {
+  slashCommandEnabled?: boolean;
```

Comment:
We already have a prop to handle disabled extensions, `disabledExtensions`, make use of that instead of creating a new prop.